lib/chore/cli.rb

```diff
@@ -266,7 +266,25 @@ def validate! #:nodoc:
         puts @parser
         exit(1)
       end
+    end
 
+    def validate_strategy!
```

Comment:
@StabbyCutyou This is another point towards needing to refactor the config support. Clunky that adding a set of strategies requires modifying the CLI class to handle validation cleanly.